### PR TITLE
added indexing scripts to docker image

### PR DESCRIPTION
**Pull request by MajoBerger**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Analysis
(Write here, if there is needed describe some specific problem. Erase it, when it is not needed.)
## Problems
(Write here, if some unexpected problems occur during solving issues. Erase it, when it is not needed.)

### CODE REVIEW

File: scripts/index-scripts/autoindexf
Comment (milanmajchrak) [inline scripts/index-scripts/autoindexf:1]: What is type of this file?

File: scripts/index-scripts/indexhandle
Comment (milanmajchrak) [inline scripts/index-scripts/indexhandle:1]: What is type of this file?

**Review by milanmajchrak — CHANGES_REQUESTED** [review]

File: scripts/index-scripts/autoindexf
Comment (MajoBerger) [reply scripts/index-scripts/autoindexf:1]: changed

File: scripts/index-scripts/indexhandle
Comment (MajoBerger) [reply scripts/index-scripts/indexhandle:1]: both

**Review by milanmajchrak — APPROVED** [review]

**Pull request merged**